menu.h

```diff
@@ -0,0 +1,103 @@
+#include <iostream>
+#include "utils.h"
+
+using namespace std;
+namespace menu
+{
+    void menu()
```

Comment:
I would name this function something more specific, like "start_menu" or something alike. There will be more "menus" down the line.

Reply:
done